content/product/design/wildcard_design_system/index.md

```diff
@@ -65,3 +65,18 @@ When a change to an existing component is required, the designer who is proposin
   - Modify the main instance of the component in Wildcard
   - Publish changes with a fitting commit message
   - Add to the "Changelog" layer in the Wildcard Figma file
+
+## User interface patterns
+
+This section will detail user interface patterns not defined in the Wildcard Design System.
+
+### Getting started pages
+
+Getting started pages are presented to viewers of major Sourcegraph feature pages when they have not seen the page or when there is no content for the section. For un-authenticated users, the page serves as an overview and prompts the user login or sign up. For authenticated users, the page serves the same purpose and also provides helpful links to start using the feature.
+
+The following rules should be applied when selecting the page to display to users:
```

Comment:
Can you add a rule about the URL schema? Something like
> The standard feature home page and the getting started view should have the same URL. For example, both the batch changes list and the batch changes getting started page are available at `/batch-changes`. Clicking the tabs at the top switches between the standard home page and the getting started page without modifying the URL.

Without this rule, it's complicated to build this functionality because it's hard to know how the user got to the page and if they should be redirected to getting started or not.

Reply:
Yes, thanks for the suggestion!